libraries/AP_GPS/AP_GPS_ERB.h

```diff
@@ -66,6 +66,13 @@ class AP_GPS_ERB : public AP_GPS_Backend
         uint8_t fix_type;
         uint8_t fix_status;
         uint8_t satellites;
+        // Introduced in ERB version 0.2.0
+        int8_t numLeapSeconds; ///< leap seconds (0xFF indicates invalid)
+        int32_t baselineN__mm; ///< distance between rover and base - North - in millimeters
```

Comment:
Why the double underbar? This actually makes it harder to type/interact with IMO.